_schemaV2.graphql

```diff
@@ -4701,6 +4701,24 @@ type HomePageRelatedArtistArtworkModule {
   basedOn: Artist
 }
 
+type IdentityVerificationType {
```

Comment:
I think the `...Type` suffix for this type is redundant (when seeing it in the SDL form here). Can probably drop that from the name.